### PR TITLE
[POT] Graph tests reference update

### PR DESCRIPTION
**Pull request by daniil-lyakhov**
### Details:
MO have changed nodes names in onnx graphs, so all onnx graph tests references was updated.
All references was checked manually 

### Tickets:
 - 86929

### CODE REVIEW

File: tools/pot/tests/test_scales.py [hunk 1]
```diff
@@ -163,8 +166,8 @@ def test_weights_scales(tmp_path, models):
         ref_min, ref_max = ref_weights[fq_name]['low_level'], ref_weights[fq_name]['high_level']
         assert_flag = False
 
-        if np.array_equal(ref_min, item_min) and \
-                np.array_equal(ref_max, item_max):
+        if np.allclose(ref_min, item_min, atol=eps) and \
+                np.allclose(ref_max, item_max, atol=eps):
```
Comment (daniil-lyakhov) [inline tools/pot/tests/test_scales.py:170]: @AlexKoff88, I've updated the way values compare to each other in `test_weights_scales`

Comment (AlexKoff88) [reply tools/pot/tests/test_scales.py:170]: I wonder where the `eps` value is coming from? can we make it lower?

Comment (daniil-lyakhov) [reply tools/pot/tests/test_scales.py:170]: I just took this value from `test_fake_quantize_configurations`. I think we could try 1e-6 value in this tests

Comment (AlexKoff88) [reply tools/pot/tests/test_scales.py:170]: I am for a lower value. It can prevent from having problems in the future.

**Comment by AlexKoff88** [comment]
I've checked several graphs (not all) and they look ok to me.

**Review by AlexKoff88 — APPROVED** [review]

**Comment by daniil-lyakhov** [comment]
Moved to https://github.com/openvinotoolkit/openvino/pull/12304

**Pull request closed without merging**